### PR TITLE
Campaign fix

### PR DESCRIPTION
**Pull request by axton21**
@angaither @DFurnes 
Adding filter and placeholder for getting empty campaigns from drupal.

### CODE REVIEW

**Comment by axton21** [comment]
@DFurnes @angaither

File: app/views/users/partials/campaigns.blade.php [hunk 1]
```diff
@@ -3,10 +3,10 @@
 		<li>
 	    <div class="figure">
 	      <div class="figure__media">
-					<img src= "{{{ $campaign['cover_image']['default']['uri'] }}}">
+					{{ isset($campaign['cover_image']['default']['uri']) ? ('<img src=' . e($campaign['cover_image']['default']['uri']) . '>') : '<img src='. (asset('assets/images/campaign-placeholder.png')) .'>' }}
```
Comment (DFurnes) [inline app/views/users/partials/campaigns.blade.php]: It might be more readable if you used a blade `@if` statement for this?

**Comment by DFurnes** [comment]
One suggestion for potentially improved readability, otherwise looks good!

**Comment by axton21** [comment]
@DFurnes cool, will just merge when builds complete

**Pull request merged**